stable/insights-chart/templates/insights-metrics-deployment.yaml

```
@@ -0,0 +1,101 @@
+# Copyright (c) 2021 Red Hat, Inc.
+# Copyright Contributors to the Open Cluster Management project
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "insights.fullname" . }}-insights-metrics
```

I find that static names are much easier to use in automation.  There's the concern of having name collision, but we haven't hit this problem with the search deployments with static names (search-operator). 
```suggestion
  name: ocm-insights-metrics
```

---

Thank you - Will Update it